### PR TITLE
make .env clickable in terminal

### PR DESCRIPTION
**Pull request by balazsorban44**
A minor chage, that makes `.env` clickable in terminals. When Next.js starts, I am showing some variables in the terminal. To jump to the .env file, adding `./` in front of it makes it clickable.

Example:
image

One disadvantage is that I am unsure how well this is supported in terminals in general (I am using VSCode's built-in one here)

Any other disadvantages?

### CODE REVIEW

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | balazsorban44/next.js patch-1 | Change |
| - | - | - | - |
| buildDuration | 11.1s | 11.4s | ⚠️ +308ms |
| nodeModulesSize | 55.3 MB | 55.3 MB | ⚠️ +2 B |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | zeit/next.js canary  | balazsorban44/next.js patch-1 | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 1.836 | 1.841 | 0 |
| / avg req/sec | 1361.52 | 1358.28 | -3.24 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 0.979 | 0.929 | -0.05 |
| /error-in-render avg req/sec | 2554.18 | 2692.3 | ⚠️ +138.12 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | balazsorban44/next.js patch-1 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.25 kB | 6.25 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..45d6.js gzip | 10.3 kB | 10.3 kB | ✓ |
| framework.a1..NSE.txt gzip | 220 B | 220 B | ✓ |
| framework.a1..NSE.txt gzip | 220 B | 220 B | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| Overall change | 56.7 kB | 56.7 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | balazsorban44/next.js patch-1 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.79 kB | 4.79 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.87 kB | 6.87 kB | ✓ |
| framework.HA..dule.js gzip | 39 kB | 39 kB | ✓ |
| Overall change | 51.4 kB | 51.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | balazsorban44/next.js patch-1 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.2 kB | 26.2 kB | ✓ |
| Overall change | 26.2 kB | 26.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | balazsorban44/next.js patch-1 | Change |
| - | - | - | - |
| _app.js gzip | 1.23 kB | 1.23 kB | ✓ |
| _error.js gzip | 3.12 kB | 3.12 kB | ✓ |
| hooks.js gzip | 663 B | 663 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.06 kB | 2.06 kB | ✓ |
| routerDirect.js gzip | 280 B | 280 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.85 kB | 7.85 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | balazsorban44/next.js patch-1 | Change |
| - | - | - | - |
| _app.module.js gzip | 598 B | 598 B | ✓ |
| _error.module.js gzip | 2.09 kB | 2.09 kB | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| index.module.js gzip | 223 B | 223 B | ✓ |
| link.module.js gzip | 1.52 kB | 1.52 kB | ✓ |
| routerDirect..dule.js gzip | 279 B | 279 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| Overall change | 5.37 kB | 5.37 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | balazsorban44/next.js patch-1 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | zeit/next.js canary  | balazsorban44/next.js patch-1 | Change |
| - | - | - | - |
| index.html gzip | 929 B | 929 B | ✓ |
| link.html gzip | 939 B | 939 B | ✓ |
| withRouter.html gzip | 926 B | 926 B | ✓ |
| Overall change | 2.79 kB | 2.79 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | zeit/next.js canary  | balazsorban44/next.js patch-1 | Change |
| - | - | - | - |
| buildDuration | 12.1s | 11.8s | -307ms |
| nodeModulesSize | 55.3 MB | 55.3 MB | ⚠️ +2 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | balazsorban44/next.js patch-1 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.25 kB | 6.25 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..45d6.js gzip | 10.3 kB | 10.3 kB | ✓ |
| framework.a1..NSE.txt gzip | 220 B | 220 B | ✓ |
| framework.a1..NSE.txt gzip | 220 B | 220 B | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| Overall change | 56.7 kB | 56.7 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | balazsorban44/next.js patch-1 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.79 kB | 4.79 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.87 kB | 6.87 kB | ✓ |
| framework.HA..dule.js gzip | 39 kB | 39 kB | ✓ |
| Overall change | 51.4 kB | 51.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | balazsorban44/next.js patch-1 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.2 kB | 26.2 kB | ✓ |
| Overall change | 26.2 kB | 26.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | balazsorban44/next.js patch-1 | Change |
| - | - | - | - |
| _app.js gzip | 1.23 kB | 1.23 kB | ✓ |
| _error.js gzip | 3.12 kB | 3.12 kB | ✓ |
| hooks.js gzip | 663 B | 663 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.06 kB | 2.06 kB | ✓ |
| routerDirect.js gzip | 280 B | 280 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.85 kB | 7.85 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | balazsorban44/next.js patch-1 | Change |
| - | - | - | - |
| _app.module.js gzip | 598 B | 598 B | ✓ |
| _error.module.js gzip | 2.09 kB | 2.09 kB | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| index.module.js gzip | 223 B | 223 B | ✓ |
| link.module.js gzip | 1.52 kB | 1.52 kB | ✓ |
| routerDirect..dule.js gzip | 279 B | 279 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| Overall change | 5.37 kB | 5.37 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | balazsorban44/next.js patch-1 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | zeit/next.js canary  | balazsorban44/next.js patch-1 | Change |
| - | - | - | - |
| _error.js | 558 kB | 558 kB | ✓ |
| 404.html | 4.18 kB | 4.18 kB | ✓ |
| hooks.html | 3.81 kB | 3.81 kB | ✓ |
| index.js | 558 kB | 558 kB | ✓ |
| link.js | 595 kB | 595 kB | ✓ |
| routerDirect.js | 587 kB | 587 kB | ✓ |
| withRouter.js | 587 kB | 587 kB | ✓ |
| Overall change | 2.89 MB | 2.89 MB | ✓ |

</details>

</details>

**Comment by timneutkens** [comment]
Potentially we can show the full path instead of saying `.env`, eg `/Users/timneutkens/projects/myproject/.env`, that way it's always clear where it came from as `./.env` is not super clear.

cc @timer @ijjk thoughts?

**Comment by balazsorban44** [comment]
I agree, if your team agrees, I can do the necessary changes

**Comment by ijjk** [comment]
## Failing test suites 

**test/integration/size-limit/test/index.test.js** 
- Production response size > should not increase the overall response size of default build

<details>
<summary>Expand output</summary>

  ● Production response size › should not increase the overall response size of default build

    expect(received).toBeLessThanOrEqual(expected)

    Expected: <= 1024
    Received:    1076

      82 |     // These numbers are without gzip compression!
      83 |     const delta = responseSizesBytes - 257 * 1024
    > 84 |     expect(delta).toBeLessThanOrEqual(1024) // don't increase size more than 1kb
         |                   ^
      85 |     expect(delta).toBeGreaterThanOrEqual(-1024) // don't decrease size more than 1kb without updating target
      86 |   })
      87 | 

      at Object.<anonymous> (integration/size-limit/test/index.test.js:84:19)

</details>

File: packages/next/lib/load-env-config.ts [hunk 1]
```diff
@@ -88,7 +88,7 @@ export function loadEnvConfig(dir: string, dev?: boolean): Env | false {
       result = dotenvExpand(result)
 
       if (result.parsed) {
-        log.info(`Loaded env from ${envFile}`)
+        log.info(`Loaded env from ./${envFile}`)
```
Comment (mohsen1) [inline packages/next/lib/load-env-config.ts:91]:
```suggestion
        log.info(`Loaded env from ${path.resolve(envFile)}`)
```

**Review by timneutkens — APPROVED** [review]

**Pull request merged**